master/buildbot/data/builds.py

```diff
@@ -20,6 +20,31 @@
 
 class Db2DataMixin(object):
 
+    def _generate_filtered_properties(self, props, filters):
+        """
+        This method returns Build's properties according to property filters.
```

Comment:
Yes that's what I meant

Le mar. 28 juil. 2015 19:27, Namgyal Brisson <notifications@github.com> a
écrit :

> In master/buildbot/data/builds.py
> <https://github.com/buildbot/buildbot/pull/1739#discussion_r35674934>:
>
> > @@ -20,6 +20,31 @@
> >
> >  class Db2DataMixin(object):
> >
> > +    def _generate_filtered_properties(self, props, filters):
> > +        """
> > +        This method returns Build's properties according to property filters.
>
> I do not see the rtype-builds.rst file in the developer section.
> Should I create it or do you mean rtype-build.rst?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buildbot/buildbot/pull/1739/files#r35674934>.
>